tools/templates/files/fileTemplates/Template Presentation Classes.kt.child.2.kt

```diff
@@ -19,7 +19,7 @@ fun ${NAME}View(
     Box(modifier, contentAlignment = Alignment.Center) {
         Text(
             "${NAME} feature view",
-            color = MaterialTheme.colorScheme.primary,
+            color = ElementTheme.colors.textprimary,
```

Comment:
This may be a typo. `textPrimary` with capital `P`?

Reply:
Good spot, thanks!